transaction-client/src/main/java/org/apache/omid/tso/client/MockTSOClient.java

```diff
@@ -46,6 +51,23 @@
     }
 
     @Override
+    public TSOFuture<Long> getFence(long tableId) {
```

Comment:
The returned transaction contains the fence timestamp as the transaction ID, documented at TransactionManager interface.
The TSO persists the fence by writing to the commit table the fence timestamp as both the start and commit timestamp.